visual-espresso/visual/src/main/java/com/saucelabs/visual/espresso/VisualCheckOptions.java

```diff
@@ -1,17 +1,45 @@
 package com.saucelabs.visual.espresso;
 
+import com.saucelabs.visual.espresso.type.RegionIn;
 import com.saucelabs.visual.espresso.utils.TestMetaInfo;
 
+import java.util.List;
+
 public class VisualCheckOptions {
     private String testName;
     private String suiteName;
+    private List<RegionIn> ignoreRegions;
 
-    public VisualCheckOptions() {
-    }
-
-    public VisualCheckOptions(String testName, String suiteName) {
+    private VisualCheckOptions(String testName, String suiteName, List<RegionIn> ignoreRegions) {
         this.testName = testName;
         this.suiteName = suiteName;
+        this.ignoreRegions = ignoreRegions;
+    }
+
+    public static class Builder {
+        private String testName;
+        private String suiteName;
+        private List<RegionIn> ignoreRegions;
+
+        public Builder withTestName(String testName) {
+            this.testName = testName;
+            return this;
+        }
+
+        public Builder withSuiteName(String suiteName) {
+            this.suiteName = suiteName;
+            return this;
+        }
+
+        public Builder withIgnoreRegions(List<RegionIn> ignoreRegions) {
```

Comment:
Should we also have a method here for passing elements and generating these regions based on the locations / coordinates from those? Is this common practice in Espresso (to assign elements to a variable to assertions on them, etc)?

Reply:
I think this might be a good idea to allow the user to pass an element and handle the coordinates by ourself